mobify/platform-scaffold
# [IM to SDK] Final cleanup

This PR moves all the remaining pieces of the integration manager into the SDK (reducer, root commands, registerConnector etc)

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-21
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/703

## Changes
- Moved connectors into their own folder
- Removed remaining pieces of the integration manager
- Updated paths to use SDK for the integration manager 

## Todos

## How to test-drive this PR
- checkout the im-to-sdk-cleanup branch for the SDK
- cd into the SDK folder
- run `npm link`
- run `npm run dev:build`
- cd back to the scaffold
- run `npm link progressive-web-sdk`
- run `npm run dev`
- preview the merlins site
- navigate throughout the site and confirm that everything works as expected
